exla/lib/exla/defn.ex

```diff
@@ -581,7 +581,7 @@ defmodule EXLA.Defn do
 
           inputs_and_shapes = Enum.reverse(reverse_inputs_and_shapes)
 
-          mode = options[:compiler_mode] || :xla
+          mode = options[:compiler_mode] || Application.get_env(:exla, :compiler_mode, :mlir)
```

Comment:
I think having just the config attribute would be fine, but if streamlines a particular use case both are fine.

Reply:
It is temporary anyway, so both are fine. :)